### ISSUE TITLE
Pretrained weight for PSPNet 

### ISSUE THREAD

**Issue by ShenZheng2000** [tengyu1998/SCI#22]
Hello, authors! I would like to know which weight version you use for PSPNet. Have you retrained or finetuned PSPNet on the training images and labels of the ACDC dataset?
Thanks!
